src/Components/Header/Header.js

```diff
@@ -0,0 +1,23 @@
+import voteImage from '../../Assets/vote.png';
+import './Header.css';
+const Header = ()  => {
+  return (
+    <div className="header-container">
+      <img 
+        className='vote-image'
+        src={voteImage}
+        alt="vote"/>
+      <p className='app-name'>My Voice, My Vote</p>
+      <button
+        className='our-story-button header-button'>
+        Our Story
+      </button>
+      <button
+        className='en-espanol-button header-button'>
+        En Español
+      </button>
+    </div>
```

Comment:
Looks like the initial className attribute is wrapped in double quotes, but then switched to single quotes.  I believe that using double quotes within JSX is the standard.

Reply:
I went through and changed all of the single quotes to double quotes across all files.